src/publicodes-state/providers/userProvider/filterLocalStorage.tsx

```diff
@@ -77,7 +77,17 @@ function handleMigrationValue({
 
   // The value is renamed and needs to be migrated
   situation[ruleName] =
-    migrationInstructions.valuesToMigrate[ruleName][nodeValue as string]
+    typeof migrationInstructions.valuesToMigrate[ruleName][
```

Comment:
Désolé je trouve ça vraiment pas très facile à lire ; contre-propale !! :
```suggestion
    const value =     migrationInstructions.valuesToMigrate[ruleName][nodeValue as string]
    if (typeof value === string && value !== 'oui' && value !== 'non') {
    situation[ruleName] = machin
    } else {
    ...
    typeof migrationInstructions.valuesToMigrate[ruleName][
```